Doc/library/__future__.rst

```diff
@@ -34,36 +40,35 @@
    where, normally, *OptionalRelease* is less than *MandatoryRelease*, and both are
    5-tuples of the same form as :data:`sys.version_info`::
 
-      (PY_MAJOR_VERSION, # the 2 in 2.1.0a3; an int
-       PY_MINOR_VERSION, # the 1; an int
-       PY_MICRO_VERSION, # the 0; an int
-       PY_RELEASE_LEVEL, # "alpha", "beta", "candidate" or "final"; string
-       PY_RELEASE_SERIAL # the 3; an int
+      (PY_MAJOR_VERSION, # the major version number,"2" in "2.1.0a3"; an integer
+       PY_MINOR_VERSION, # the minor version number,"1" in "2.1.0a3"; an integer
+       PY_MICRO_VERSION, # the micro version number,"0" in "2.1.0a3"; an integer
+       PY_RELEASE_LEVEL, # "alpha", "beta", "candidate", or "final"; a string
+       PY_RELEASE_SERIAL, # the release serial number,"3" in "2.1.0a3"; an integer
       )
 
+The release information for features is captured through *OptionalRelease* and *MandatoryRelease*.
+The *OptionalRelease* records the first release in which the feature was accepted, and
+*MandatoryRelease* records when the feature became part of the language. Once a feature is included
+in a *MandatoryRelease*, modules no longer need a future statement to use the feature, but they may
+continue to use such imports if needed.
```

Comment:
Please wrap text to max 80 characters.

https://devguide.python.org/documentation/style-guide/#use-of-whitespace